### ISSUE TITLE
Imaging: papers

### ISSUE THREAD

**Issue by twang15** [twang15/Feature-selection#10]
## interpretability
0. [2020-Nature Machine Intelligence-From local explanations to global understanding with explainable AI for trees](https://www.nature.com/articles/s42256-019-0138-9.epdf?shared_access_token=RCYPTVkiECUmc0CccSMgXtRgN0jAjWel9jnR3ZoTv0O81kV8DqPb2VXSseRmof0Pl8YSOZy4FHz5vMc3xsxcX6uT10EzEoWo7B-nZQAHJJvBYhQJTT1LnJmpsa48nlgUWrMkThFrEIvZstjQ7Xdc5g%3D%3D)
* **Explanation for CKD progression**
* **A must read**
1. [2017-NIPS-S. Lundberg, S.-I. Lee, A Unified Approach to Interpreting Model Predictions, Adv. Neural Inf. Process. Syst. 2017-Decem (2017) 4766–4775.](https://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions)
* Shapley value
* the summary is that a proof from game theory on the fair allocation of profits leads to a uniqueness result for feature attribution methods in machine learning. These unique values are called Shapley values, after Lloyd Shapley who derived them in the 1950’s. The SHAP values we use here result from a unification of several individualized model interpretation methods connected to Shapley values.
2. [2010-UCB-To-Explain-or-to-Predict](https://projecteuclid.org/journals/statistical-science/volume-25/issue-3/To-Explain-or-to-Predict/10.1214/10-STS330.full)
3. Leeper, T.J., (2017). Interpreting regression results using average marginal effects with R’s margins. Tech. rep.
* Marginal effects
4. [2016-KDD-"Why Should I Trust You?": Explaining the Predictions of Any Classifier](https://dl.acm.org/doi/10.1145/2939672.2939778)
* Locally Interpretable Model-Agnostic Explanations
* an algorithm that can explain the predictions of any classifier or regressor in a faithful way, by approximating it locally with an interpretable model
* [code, video, slides, tutorial](https://github.com/marcotcr/lime)
5. [2018-Nature Biomedical-An explainable deep-learning algorithm for the detection of acute intracranial haemorrhage from small datasets](https://www.nature.com/articles/s41551-018-0324-9)
6. [2018-Nature Biomedical-Explainable machine-learning predictions for the prevention of hypoxaemia during surgery](https://www.nature.com/articles/s41551-018-0304-0)

## Causal reasoning and machine learning
1. [2021-Towards Causal Representation Learning](https://arxiv.org/abs/2102.11107)
* [Talk video: 2020-Yoshua Bengio-Towards causal representation learning: An AI & Deep learning Perspective on Causality](https://www.youtube.com/watch?v=rKZJ0TJWvTk)
2. [2019-Causality for Machine Learning](https://arxiv.org/abs/1911.10500)
3. [2018-KDD-Tutorial on Causal Inference and Counterfactual Reasoning](https://causalinference.gitlab.io/kdd-tutorial/)
4. [2020-MSR-DoWhy: An End-to-End Library for Causal Inference](https://arxiv.org/abs/2011.04216)
* [slides, code, and tutorial](https://github.com/microsoft/dowhy)
5. [2020-MSR-EconML](https://github.com/microsoft/econml)
* Shap values
* combine state-of-the-art machine learning techniques with econometrics to bring automation to complex causal inference problems

## Model performance
1. [2016-XGBoost: A Scalable Tree Boosting System](https://arxiv.org/abs/1603.02754)
2. [2021-Cortex: A Compiler for Recursive Deep Learning Models](https://arxiv.org/abs/2011.01383)

### AI in medicine
1. [2021-Nature-Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans](https://www.nature.com/articles/s42256-021-00307-0)
2. [2019-Nature Medicine-End-to-end lung cancer screening with three-dimensional deep learning on low-dose chest computed tomography](https://www.nature.com/articles/s41591-019-0447-x)
3. [Automated Chest X-Ray Interpretation](https://glassboxmedicine.com/2019/05/11/automated-chest-x-ray-interpretation/)

[When to Impute? Imputation before and during cross-validation](https://arxiv.org/abs/2010.00718?context=stat.CO)

### Interpretability for Deep Neural Network
1. 2020-12-A Survey on Neural Network Interpretability
* Chinese review: https://zhuanlan.zhihu.com/p/368755357
* Knowledge Graph for Interpretability: https://zhuanlan.zhihu.com/p/386458680
2. Paul Allison
* Prediction vs. Causation in Regression Analysis: https://statisticalhorizons.com/prediction-vs-causation-in-regression-analysis
* 
[2014-Paul Alison-Prediction vs. Causation in Regression Analysis _ Statistical Horizons.pdf](https://github.com/CYZQ2015/ckd-study/files/7035446/2014-Paul.Alison-Prediction.vs.Causation.in.Regression.Analysis._.Statistical.Horizons.pdf)

3. **Inference and Prediction comparison**:
* https://www.youtube.com/watch?v=uh_k1jD35K8&list=RDCMUCKPyg5gsnt6h0aA8EBw3i6A&index=1

R. Kohavi et al., A Study of Cross-validation and Bootstrap for Accuracy Estimation and Model Selection, International Joint Conference on Artificial Intelligence (IJCAI), 14(12), Seiten 1137–1145, 1995
* 10-fold cross-validation is the best compromise between bias and variance in most cases

[2010-Journal of Machine learning research-On Over-fitting in Model Selection and Subsequent Selection Bias in
Performance Evaluation](https://www.jmlr.org/papers/volume11/cawley10a/cawley10a.pdf)
* this study concentrates on cross-validation based model selection, the findings are quite general and apply to any model
selection practice involving the optimisation of a model selection criterion evaluated over a finite
sample of data, including maximisation of the Bayesian evidence and optimisation of performance
bounds
